examples/Cluster_update_examples.ipynb

```diff
@@ -0,0 +1,3300 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    plt.errorbar(betas, ene_cluster, fmt='--o', label='cluster')
</pre><p><code>plt.errorbar(x, y, yerr=...)</code> to actually show the errorbars</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ev-br/mc_lib/pull/22/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>